packages/config/config/devices/0x010f/fgms001.json

```diff
@@ -550,7 +592,7 @@
 			"#": "64",
 			"$if": "firmwareVersion >= 3.2",
 			"label": "Temperature report - interval",
-			"description": "How often the temperature reports will be sent",
+			"description": "Time interval (seconds) between consecutive temperature reports (sent even when there is no change).",
```

Comment:
The unit should be specified using the `"unit"` property, not as part of the description.